contracts/liquidityMining/KyberRewardLocker.sol

```diff
@@ -221,38 +211,43 @@ contract KyberRewardLocker is IKyberRewardLocker, PermissionAdmin {
   {
     VestingSchedules storage schedules = accountVestingSchedules[msg.sender][token];
     uint256 totalVesting = 0;
-    uint256 totalSlashing = 0;
     for (uint256 i = 0; i < indexes.length; i++) {
       VestingSchedule memory schedule = schedules.data[indexes[i]];
       if (schedule.quantity == 0) {
         continue;
       }
       uint256 vestQuantity = _getVestingQuantity(
         schedule.quantity,
-        schedule.startTime,
-        schedule.endTime
+        schedule.startBlock,
```

Comment:
line 216 can be removed, let `_getVestingQuantity` handle when quantity is 0

Reply:
updated